landing_page/spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+    context "create a new user account with acceptable credentials" do
+
+        it "successfully adds a new user" do
+            expect{create_new_user}.to change(User, :count).by(1)
+        end
+
+        it "logs the user in upon account creation" do
+            session_id  = session[:user_id]
```

Comment:
This is confusing. How is it there's a `session[:user_id]`?

Reply:
Resolved in commit 0a7fed3763eb9f6d8481ed307599325f2e63d1cd